### PR TITLE
python version change for aws

### PR DESCRIPTION
**Pull request by sagarnavale**

### CODE REVIEW

File: tests/data/gcp/function.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "description": "Cloud Function123",
         "status": "ACTIVE",
         "entryPoint": "function123.cloud.google.com",
-        "runtime": "python3.8",
```
Comment (mpurusottamc) [inline tests/data/gcp/function.py:8]: We should not change GCP function related files. Requirement is only for AWS.

File: Dockerfile [hunk 2]
```diff
@@ -4,13 +4,13 @@ WORKDIR /srv/cartography
 
 ENV PATH=/venv/bin:$PATH
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3.8-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev curl && \
```
Comment (mpurusottamc) [inline Dockerfile:7]: Let's revert back all the changes in this file. We should not change `Dockerfile`. We should change only Cloudanix specific files.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
minor changes are needed.

**Comment by sagarnavale** [comment]
all changes are Done

**Comment by sagarnavale** [comment]
changes are Done

**Pull request merged**